pkg/imageserver/webdav.go

```diff
@@ -13,31 +13,30 @@ import (
 )
 
 const (
-	// CHROOT_SERVE_PATH is the path to server if we are performing a chroot
+	// chrootServePath is the path to server if we are performing a chroot
 	// this probably does not belong here.
-	CHROOT_SERVE_PATH = "/"
-	// AUTH_TOKEN_HEADER is the custom HTTP Header used
+	chrootServePath = "/"
+	// authTokenHeader is the custom HTTP Header used
 	// to authenticate to image inspector.
-	// We use a custom auth header instead of Authorization
+	// Clients must use a custom auth header
+	// instead of standard Authorization
 	// because Kubernetes Proxy strips the default Auth Header
 	// from requests
-	AUTH_TOKEN_HEADER = "X-Auth-Token"
+	authTokenHeader = "X-Auth-Token"
 )
 
 // webdavImageServer implements ImageServer.
 type webdavImageServer struct {
-	opts   ImageServerOptions
-	chroot bool
+	opts ImageServerOptions
 }
 
 // ensures this always implements the interface or fail compilation.
 var _ ImageServer = &webdavImageServer{}
 
 // NewWebdavImageServer creates a new webdav image server.
-func NewWebdavImageServer(opts ImageServerOptions, chroot bool) ImageServer {
+func NewWebdavImageServer(opts ImageServerOptions) *webdavImageServer {
```

Comment:
yes, usually the tests would be in the same package so it could use the private type if it needed to.  Anything other than a unit test probably shouldn't care about the actual type of `ImageServer` though since it's an implementation detail if it isn't configurable or settable via injection.  If it does then maybe we're missing some interface methods somewhere that would be helpful